rst.html.markdown

```diff
@@ -72,36 +72,36 @@ France      Paris
 Japan       Tokyo
 =========== ========
 
-More complexe tabless can be done easily (merged columns and/or rows) but I suggest you to read the complete doc for this :)
+More complex tabless can be done easily (merged columns and/or rows) but I suggest you to read the complete doc for this :)
 
-Their is multiple ways to make links:
+There are multiple ways to make links:
 
-- By adding an underscore after a word : Github_ and by adding the target after the text (this have the advantage to not insert un-necessary URL inside the readed text).
-- By typing a full comprehensible URL : https://github.com/ (will be automatically converted in link)
-- By making a more "markdown" link: `Github <https://github.com/>`_ .
+- By adding an underscore after a word : Github_ and by adding the target URL after the text (this way has the advantage to not insert unnecessary URLs inside readable text).
+- By typing a full comprehensible URL : https://github.com/ (will be automatically converted to a link)
+- By making a more Markdown-like link: `Github <https://github.com/>`_ .
 
 .. _Github https://github.com/
 
 ```
 
 
-## How to use it
+## How to Use It
 
-RST comes with docutils in which you have `rst2html` for exemple:
+RST comes with docutils where you have `rst2html`, for example:
 
 ```bash
 $ rst2html myfile.rst output.html
 ```
 
 *Note : On some systems the command could be rst2html.py*
 
-But their is more complexe applications that uses RST file format:
+But there are more complex applications that use the RST format:
 
 - [Pelican](http://blog.getpelican.com/), a static site generator
 - [Sphinx](http://sphinx-doc.org/), a documentation generator
 - and many others
 
 
-## Readings
+## Reading
```

Comment:
@sidgupta234, OK, will revert. Just never seen "readings" in any LearnXInYMinutes files.